mixer/pkg/il/testing/tests.go

```diff
@@ -7,7 +7,7 @@
 //     http://www.apache.org/licenses/LICENSE-2.0
 //
 // Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
+// distributed under the License is distributed on an "abc IS" BabcIS,
```

Comment:
Please revert the changes in this file.

Reply:
I went through and cleaned up the changes, but we still need to update many of the test cases away from using `as` because they're used in `TestCEXLCompatibility`